src/lib/temple/reset.ts

```diff
@@ -11,6 +12,7 @@ export async function clearAsyncStorages() {
   await Repo.db.delete();
   await Repo.db.open();
   await browser.storage.local.clear();
+  await browserWithSessionStorage.storage.session?.clear();
```

Comment:
maybe create `clearSessionStorage` method inside `src/lib/temple/back/vault/session-store.ts`?   
All code related to `browser.storage.session` will be there